client/my-sites/checkout/composite-checkout/components/wp-contact-form.js

```diff
@@ -269,9 +268,7 @@ function DomainContactDetails( {
 		! hasDomainRegistration( responseCart ) &&
 		! hasTransferProduct( responseCart );
 	const getIsFieldDisabled = () => isDisabled;
-	const needsAlternateEmailForGSuite =
-		needsOnlyGoogleAppsDetails &&
-		needsExplicitAlternateEmailForGSuite( responseCart, contactDetails );
+	const needsAlternateEmailForGSuite = needsOnlyGoogleAppsDetails;
```

Comment:
Same remark, we could remove that variable and pass `needsOnlyGoogleAppsDetails` directly.